packages/shared/src/DateTimeHelpers/DateTimeHelpers.ts

```diff
@@ -0,0 +1,229 @@
+  const _dt = dt instanceof DateTime
+    ? dt
+    : dt instanceof Date
+      ? DateTime.fromJSDate(dt, zoneOption).setLocale(getLocaleAsString(locale || ''))
+      : typeof dt === 'string'
+        ? DateTime.fromISO(dt, zoneOption).setLocale(getLocaleAsString(locale || ''))
+        : DateTime.fromObject(dt, zoneOption).setLocale(getLocaleAsString(locale || ''))
+  if (_dt.invalidReason) {
+    throw new Error(`Invalid date: ${_dt.invalidReason}`)
+  }
+  return _dt
+}
+
+const isSame = (
+  dt1: string | Date | DateTime | Object,
+  dt2: string | Date | DateTime | Object,
+  unit: DateTimeUnit,
+  locale?: string
+): boolean => {
+  const _dt1 = toDateTime(dt1, { locale })
+  const _dt2 = toDateTime(dt2, { locale })
+  return _dt1.hasSame(_dt2, unit)
+}
+
+export const rangeIncludesDate = (
+  start: string | Date | DateTime | Object,
+  date: string | Date | DateTime | Object,
+  end: string | Date | DateTime | Object,
+): boolean => {
+  const _start = toDateTime(start)
+  const _date = toDateTime(date)
+  const _end = toDateTime(end)
+  return _date.hasSame(_start, 'day')
+    || _date.hasSame(_end, 'day')
+    || (_start < _date && _date < _end)
+}
+
+export const inWeek = (
+  start: string | Date | DateTime | Object,
+  date: string | Date | DateTime | Object,
+  end: string | Date | DateTime | Object
+): boolean => {
+  const _start = toDateTime(start)
+  const _date = toDateTime(date)
+  const _end = toDateTime(end)
+  const weekStart = _date.startOf('week', { useLocaleWeeks: true })
+  const weekEnd = _date.endOf('week', { useLocaleWeeks: true }).plus({ days: 1 })
+  return _start < weekEnd && weekStart <= _end
+}
+
+export const isSameDay = (
+  date1 : string | Date | DateTime | Object,
+  date2 : string | Date | DateTime | Object
+) : boolean => {
+  return isSame(date1, date2, 'day')
+}
+
+export const isSameWeek = (
+  date1 : string | Date | DateTime | Object,
+  date2 : string | Date | DateTime | Object
+) : boolean => {
+  return isSame(date1, date2, 'week')
+}
+
+export const isSameMonth = (
+  date1 : string | Date | DateTime | Object,
+  date2 : string | Date | DateTime | Object
+) : boolean => {
+  return isSame(date1, date2, 'month')
+}
+
+export function humanDate (
+  date: string | Date | DateTime | Object,
+  short?: boolean,
+  locale?: string
+): string {
+  const _date = toDateTime(date, { locale })
+  let ret = _date.invalidReason ? '' : prettyDate.format(_date.toJSDate())
+
+  // Always return 'now' for very recent timestamps
+  if (ret === 'just now') {
+    return 'now'
+  }
+
+  if (short) {
+    ret = ret.replace(' ago', '')
+  } else {
+    if (ret.match(/(\d+) seconds? ago/)) {
+      return 'now'
+    }
+  }
+
+  return ret.replace(/ seconds?/, 's')
+    .replace(/ minutes?/, 'm')
+    .replace(/ hours?/, 'h')
+    .replace(/ days?/, 'd')
+    .replace(/ weeks?/, 'w')
+    .replace(/ years?/, 'y')
+    .replace(/ month(s?)/, ' mo$1')
+}
+
+export const formatDatePair = (
```

Comment:
There have definitely been tweaks to formatDatePair recently so we have to make sure this is up to date with the latest

Reply:
Ok, I have now synced the two versions of formatDatePair, switched all of their use to one of them, changed the param signature to be an object (there were five or six params, was getting hella nasty) and then nuked the deprecated one.